lib/web_ui/dev/felt_windows.bat

```diff
@@ -38,13 +38,6 @@ SET SNAPSHOT_PATH="%DART_TOOL_DIR%felt.snapshot"
 CD %FLUTTER_DIR%
 FOR /F "tokens=1 delims=:" %%a in ('git rev-parse HEAD') DO SET REVISION=%%a
 
-:: Uncomment for debugging the values.

```

Comment:
Add TODO: Update LUCI to use felt.bat and remove.

Reply:
Done.